__tests__/commands/_helpers.js

```diff
@@ -51,6 +52,10 @@ export async function getPackageVersion(config: Config, packagePath: string): Pr
   return json.version;
 }
 
+export function getTempGlobalFolder(): string {
+  return path.join(os.tmpdir(), `yarn-global-${Math.random()}`);
+}
```

Comment:
@zurawiki I suggest to you to use `import mkdir from './_temp.js';`. One example is in https://github.com/yarnpkg/yarn/blob/eac1836cea4a2a8e89d140fa22c204c1c416f14a/__tests__/fetchers.js#L10